keystorage/build.gradle

```diff
@@ -0,0 +1 @@
+jar { enabled = false }
```

Comment:
whats this about?

Reply:
Its something we should have always had - i.e. we should not create a jar file at this level of the project (no, it shouldn't be in THIS pr) - but it does need to go in sometime.